streams/jvm/src/test/scala/zio/stream/SinkSpec.scala

```diff
@@ -225,4 +228,55 @@ class SinkSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
       } yield test
     }
   }
+
+  private def throttleShape = {
+
+    def sinkTest(sink: ZSink[Clock, Nothing, Nothing, Int, Int]) =
```

Comment:
very nice use of the TestClock!

Reply:
Thanks. And thank you for the help with the sbt setup.